### PR TITLE
K4B-2937 Update PAPI docs

### PR DESCRIPTION
**Pull request by maxno-kivra**
This fixes two issues with the documentation for PAPI. First off, when updating the labels of a content you must nest them under `labels`. Secondly, the interpreter has been deprecated and this marks all related fields as such and removes a paragraph about it.

---

This does some _small_ cleanup (the previous version of this PR did too much). Now it's just removing trailing whitespace because my editor does that, and fixing the text/csv example.

### CODE REVIEW

**Review by kivra-fabber — CHANGES_REQUESTED** [review]
Please separate the reorganisation to a separate PR. I cannot review the fixes to the labels and the interpreter in a diff like this.

**Comment by maxno-kivra** [comment]
The split is now in #185.

**Review by kivra-fabber — APPROVED** [review]
Nice!

File: swagger.yml [hunk 1]
```diff
@@ -830,8 +830,12 @@ tags:
...
   - name: "Tenant API - Agreements"
     description: Endpoints for managing signatures and agreements
-  - name: "Partner API"
-    description: Endpoints for partner access to company mailboxes
+  - name: "Partner API v1"
+    description: |
+      Endpoints for partner access to company mailboxes.
+  - name: "Partner API v3"
+    description: |
+      Endpoints for partner access to company mailboxes.
```
Comment (kivra-fabber) [inline swagger.yml:838]: Good catch 👍

File: swagger.yml [hunk 2]
```diff
@@ -3033,11 +3035,11 @@ paths:
...
               schema:
-                example: |
-                  key,type,name,vat_number
-                  2156077433683911cdca5fd4563bded979572454cb9,company,Cornelias
-                  Café AB,SE556000475501
                 type: string
+              example: |
+                key,type,name,vat_number
+                2156077433683911cdca5fd4563bded979572454cb9,company,Cornelias
+                Café AB,SE556000475501
```
Comment (kivra-fabber) [inline swagger.yml:3042]: also good catch ⭐

**Pull request merged**